services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -122,27 +137,28 @@ class ApplicationAttachmentService(
         }
     }
 
-    /**
-     * Attachment should be sent if application is in Allu (alluId present) but not yet in handling.
-     */
-    private fun sendIfPending(application: Application, attachment: ApplicationAttachmentEntity) =
-        with(application) {
-            logger.info { "Check application if should send attachment, alluId: '$alluid'" }
-            if (alluid != null && sendable(alluid, alluStatus)) {
-                cableReportService.addAttachment(alluid, attachment.toAlluAttachment())
-            }
-        }
+    private fun gonePastHandling(application: ApplicationEntity): Boolean = !isPending(application)
```

Comment:
I think `!isPending` is both more accurate and easier to understand than `gonePastHandling`.

Using different names for the same things between ApplicationService and ApplicationAttachmentService is unnecessarily confusing.

Reply:
Changed as suggested.